### DISCUSSION TITLE
Suricata HOME_NET

### DISCUSSION THREAD

**Discussion by V1D1AN** [telekom-security/tpotce#909] [Q&A]
When you install T-POT , the default HOME_NET of suricata is:

```
HOME_NET: "[192.168.0.0/16,10.0.0.0/8,172.16.0.0/12]"
```

Can you integrate in the installation of TPOT , the modification of the HOME_NET for Suricata.

Example:

```
HOME_NET:"[XXX.XXX.XXX.XXX/24]" 
```

Sincerely

**Comment by t3chn0m4g3** [comment]
You can adjust the configuration to your liking by using [docker volumes](https://github.com/telekom-security/tpotce/wiki/Reconfigure-ews.cfg). An example is provided in the Wiki.

**Reply by StevenD33** [reply]
by default suricata is sending false alarm and alert becasue the home net is not configured. It could be better to automate this task during installation
